lib/rentlinx/validators/address_validator.rb

```diff
@@ -0,0 +1,10 @@
+module Rentlinx
+  class AddressValidator < BaseValidator
```

Comment:
This needs documentation. You're a little ways behind master, remember to pull before starting a new branch.